app/views/templates/tests-by-category-widget.html.slim

```diff
@@ -0,0 +1,66 @@
+.panel.panel-default
+  .panel-heading
+    .row
+      .col-md-6
+        h3.panel-title Tests by category
+      .col-md-6
+        div style='float:right;'
+          a.btn.btn-default.btn-xs href='#' role='button'
+            i.glyphicon.glyphicon-filter
+            drop constrain-to-scroll-parent='true' constrain-to-window='true' open-on='"click"' position='"left top"' tether-options='{offset: "-5px -5px"}' before-close='beforeClose()'
+              .panel.panel-default.tests-by-category-filters
+                .panel-heading
+                  .h4.panel-title Filters
+                .panel-body
+                  form.form role='form'
+                    .col-md-12 ng-if='!project'
+                      project-select prefix='tests-by-category' organization='organization' model-object='params'
+                    .col-md-12 ng-if='project'
+                      project-version-select prefix='tests-by-category' organization='organization' project='project' model-object='params' auto-select='true'
+                    .col-md-12
+                      user-select prefix='tests-by-category' organization='organization' model-object='params'
+                    .col-md-12
+                      .btn-group.btn-group-justified
+                        .btn-group
+                          button.btn.btn-warning ng-click='resetFilters()' Reset
+                        .btn-group
+                          button.btn.btn-primary ng-click='closeFilters()' Close
+
+  .panel-body ng-if='testsByCategory.length === 0'
+    .empty
+      em Loading...
+  .panel-body ng-if='testsByCategory.length > 0'
+    table.table.tests-by-category-table
```

Comment:
It's a bad idea to use Bootstrap's `.table` here if you're going to override all its styles so that it looks nothing like Bootstrap's table. Simply use a normal table so that we only have to add the custom style we need.